docs/replication-manager/replication-manager-for-pxc.md

```diff
@@ -0,0 +1,307 @@
+wsrep_cluster_address=gcomm://10.0.4.160,10.0.4.162,10.0.4.163
+wsrep_replica_threads= 2
+wsrep_log_conflicts
+wsrep_cluster_name=DC1
+pxc_strict_mode=ENFORCING
+wsrep_sst_method=xtrabackup-v2
+wsrep_sst_auth="root:root"
+```
+
+All nodes have the same `server-id` value. In a multi-source replication topology, the replicas require the `source_info_repository` and `relay_log_info_repository` values to be `TABLE`. These settings cannot be `FILE`.
+
+### Configuration steps
+
+1. Bootstrap the cluster on DC1-1 node by running the following commands as root or using sudo:
+
+    ```bash
+    [root@DC1-1 ~]# /etc/init.d/mysql stop 
+    [root@DC1-1 ~]# /etc/init.d/mysql bootstrap-pxc
+    ```
+
+2. On DC1-2 node, stop MySQL, delete the content of the datadir to force the SST, and start MySQL. Run the following commands as root or using sudo:
+
+    ```bash
+    [root@DC1-2 ~]# /etc/init.d/mysql stop
+    [root@DC1-2 ~]# rm -rf /var/lib/mysql/*
+    [root@DC1-2 ~]# /etc/init.d/mysql start
+    ```
+
+3. Once the SST of DC1-2 node is completed, proceed on DC1-3. Run the following commands as root or using sudo:
+
+    ```bash
+    [root@DC1-3 ~]# /etc/init.d/mysql stop
+    [root@DC1-3 ~]# rm -rf /var/lib/mysql/*
+    [root@DC1-3 ~]# /etc/init.d/mysql start
+    ```
+
+4. The DC1 cluster uses a single [GTID](../glossary.md#gtid) sequence. To make sure `GTID_PURGED` is the same, run the following commands on DC1-1, DC1-2, and DC1-3 nodes:
+
+    ```bash
+    mysql> flush logs;
+    mysql> purge source logs to 'mysql-bin.000003';
+    ```
+
+    `mysql-bin.000003` is the last file returned from the show source logs. 
+
+5. Adjust the aformentioned steps to set up the DC2 and DC3 clusters accordingly.
+    
+6. Add privileges for replication between all nodes only for DC1: 
+
+    ```bash
+    GRANT REPLICATION CLIENT, REPLICATION REPLICA ON *.* TO 'repl'@'%' identified by 'replpass';
+    ```
+
+7. Add the replication tables. For example, we add the tables to DC1: 
+    
+    ```text
+    create database if not exists percona;
+    use percona;
+    CREATE TABLE `replication` (
+      `host` varchar(40) NOT NULL,
+      `weight` int(11) NOT NULL DEFAULT 0,
+      `localIndex` int(11) DEFAULT NULL,
+      `isReplica` enum('No','Yes','Proposed','Failed') DEFAULT 'No',
+      `lastUpdate` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+      `lastHeartbeat` timestamp NOT NULL DEFAULT '1971-01-01 00:00:00',
+      `connectionName` varchar(64) NOT NULL,
+      PRIMARY KEY (`connectionName`,`host`),
+      KEY `idx_host` (`host`)
+    ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+    CREATE TABLE `cluster` (
+      `cluster` varchar(31) NOT NULL,
+      `sourceCandidates` varchar(255) NOT NULL,
+      `replCreds` varchar(255) NOT NULL,
+      PRIMARY KEY (`cluster`)
+    ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+    CREATE TABLE `link` (
+      `clusterReplica` varchar(31) NOT NULL,
+      `clusterSource` varchar(31) NOT NULL,
```

Comment:
Please bring back 'clusterMaster' and 'clusterSlave'  instead of clusterSource and clusterReplica as this breaks the script

Reply:
Updated